frontend/src/components/Catalog.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from 'react';
+import {
+  Card,
+  CardImg,
+  CardText,
+  CardBody,
+  CardTitle,
+  CardSubtitle,
+  Button
+} from 'reactstrap';
+import { Container, Row, Col } from 'reactstrap';
+import PropTypes from 'prop-types';
+import { Pagination, PaginationItem, PaginationLink } from 'reactstrap';
+
+class Catalog extends Component {
+  state = {
+    page: 1
+  };
+
+  getFirstRow = () => {
+    var page = this.state.page;
+    var start = (page - 1) * 6;
+    return this.props.cards.slice(start, start + 3);
```

Comment:
nit: avoid use of magic numbers such as 6, 3 and declare as variables or accept as props